sfaira/data/dataloaders/base/dataset_group.py

```diff
@@ -486,7 +486,7 @@ def doi(self) -> List[str]:
         """
         dois = []
         for _, v in self.datasets.items():
-            vdoi = v.doi_journal
+            vdoi = v.doi_journal if v.doi_journal is not None else v.doi_preprint
```

Comment:
Is this not equivalent to `vdoi = v.doi`? That would be preferable as it pushes DOI management into the Dataset class and thus makes it more centralized.

Reply:
of course, that make so much more sense!